tests/test_interface_switchport.rb

```diff
@@ -427,7 +427,7 @@ def test_switchport_autostate_enabled_access
            'switchport autostate exclude')
 
     cmd_ref = cmd_ref_switchport_autostate_exclude
-    if cmd_ref.config_set?
+    if cmd_ref.setter?
```

Comment:
Glenn added ``setter?`` to command_reference.rb as part of the yaml refactoring.